### ISSUE TITLE
bug: wit/contact entity causes "Something went wrong. We've been notified" when language is set to Polish

### ISSUE THREAD

**Issue by dominik-mukrecki** [wit-ai/wit#2450]
It seems like `wit/contact` bulit-in entity causes a bug for the Polish language.

With this entity turned on an error appears in the API response after `GET https://api.wit.ai/message` call:

`Something went wrong. We've been notified`

Turing this entity off fixes API response.

The same error appears currently in my Meta App, when NLP is turned on, regardless of language or model. Due to this I am forced to provide a workaround for my Messenger-based chat app.

In short: NLP built-in into the Meta App does not work at all, but using API directly with the fix mentioned above works for now.

**Comment by nishsinghal20** [comment]
Hey @dominikmukrecki! Can you share the app ID of your app so we can look further into it?

**Comment by Barbog** [comment]
Closing due to no movement on the issue. Please re-open or file a new task should the issue be persisting.
